entity_meta_relation.routing.yml

```diff
@@ -0,0 +1,7 @@
+emr.admin_structure:
+  path: '/admin/structure/emr'
+  defaults:
+    _controller: '\Drupal\system\Controller\SystemController::systemAdminMenuBlockPage'
+    _title: 'Entity meta relation'
+  requirements:
+    _permission: 'access administration pages'
```

Comment:
Let's use the permission that is needed to access entities of this kind no? You can specify multiple permissions.

Reply:
@hernani You should specify also the `access entity meta overview` permission in an OR way here. Either or.

Reply:
I think the comment is outdated.